pkgs/development/libraries/nanoflann/default.nix

```diff
@@ -0,0 +1,22 @@
+{stdenv, fetchurl, cmake}:
+
+stdenv.mkDerivation {
+  name = "nanoflann-1.1.9";
+
+  src = fetchurl {
```

Comment:
github's generated tarballs tend to change, please use `fetchFromGitHub` instead which derives the hash from the archive's contents.